pkg/apis/core/types.go

```diff
@@ -992,6 +992,40 @@ type CinderVolumeSource struct {
 	// the ReadOnly setting in VolumeMounts.
 	// +optional
 	ReadOnly bool
+	// Optional: points to a secret object that has enough information to connect
```

Comment:
I would recommend leaving this more generic ("Optional: points to a secret object containing parameters used to connect to OpenStack") until the issues raised in https://github.com/kubernetes/kubernetes/pull/61295#discussion_r187440285 are resolved (which it sounds like will be done separately and possibly externally)